steps/github-step-repositories-describe/step.yaml

```diff
@@ -0,0 +1,60 @@
+# The schema version. Required. Must be exactly the string "integration/v1".
+apiVersion: integration/v1
+
+# The schema kind. Required. Must be one of "Query", "Step", or "Trigger"
+# corresponding to its directory location.
+kind: Step
+
+# The name of the action. Required. Must be exactly the name of the directory
+# containing the action.
+name: github-step-repositories-describe
+
+# The version of the action. Required. Must be an integer. If specified in the
+# directory name, must be exactly the version in the directory name.
+version: 1
+
+# High-level phrase describing what this action does. Required.
+summary: Describe GitHub repositories belonging to an org
+
+# Single-paragraph explanation of what this action does in more detail.
+# Optional. Markdown.
+description: |
+  Describe GitHub repositories belonging to an org that match given parameters.
+# URL or path relative to this file to an icon or icons representing this
+# action. Optional. Defaults to the integration icon.
+icon:
+
+# The mechanism to use to construct this step. Required. Must be an action
+# builder. See the Builders section below.
+build:
+  # The schema version for builders. Required. For now, must be the exact
+  # string "build/v1". We may consider supporting custom third-party builders
+  # in the future.
+  apiVersion: build/v1
+
+  # The builder to use. Required.
+  kind: Docker
+
+publish:
+  repository: relaysh/relay-github
+
+schemas:
+  spec:
+    source: file
+    file: spec.schema.json
+  outputs:
+    source: file
+    file: outputs.schema.json
+
+examples:
+- summary: Describe GitHub org repositories
+  content:
+    apiVersion: v1
+    kind: Step
+    name: describe-images
+    image: relaysh/github-step-repositories-describe
+    spec:
+      github:
+        connection: !Connection { type: github, name: my-github-account }
```

Comment:
This YAML tag syntax is quite old now, but anyway, vis-a-vis the previous comment about the connection not being ready, this should probably be something like:

```yaml
connection:
  accessToken: ${secrets.github-access-token}
```

Reply:
lol, I knew you'd ding me for that